zipline/testing/core.py

```diff
@@ -748,6 +748,9 @@ def get_spot_value(self, asset, field, dt, data_frequency):
         # otherwise just return a fixed value
         return int(asset)
 
+    # XXX: These aren't actually the methods that are used by the superclasses,
+    # so these don't do anything, and this class will likely produce unexpected
+    # results for history().
     def _get_daily_window_for_sid(self, asset, field, days_in_window,
```

Comment:
I figured the original intent here would be more helpful for whoever eventually gets bitten by this.  I just happened to notice that these aren't used because I was looking at renaming methods in data portal.